includes/Integrations/Core_Themes_Support.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Class Core_Themes_Support
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Customizer;
+
+/**
+ * Class Core_Themes_Support.
+ */
+class Core_Themes_Support {
+
+	/**
+	 * Default array of core themes to add support to.
+	 *
+	 * @var array
+	 */
+	public static $supported_themes = [
+		'twentytwentyone',
+		'twentytwenty',
+		'twentynineteen',
+		'twentyseventeen',
+		'twentysixteen',
+		'twentyfifteen',
+		'twentyfourteen',
+		'twentythirteen',
+		'twentytwelve',
+		'twentyeleven',
+		'twentyten',
+		'classic',
+		'default',
```

Comment:
Whoa, those were the times! Fun to see that this would work. Do you have a screenshot?

Given that these themes are so old and not really used these days, I don't think we really need those integrations (even though it's cool) :-)

Reply:
the tests are underway, and CSS fixes will be done in a separate PR. As you mentioned even though it's cool, but if we do not need it then I would recommend to remove those themes.

Which ones do you think are not required?

Reply:
I would say the default and classic themes are not heavily used. But I see lots of twenty ten themed around. I wonder if there are any stats online for themes..

Reply:
@divyarajmasani The ones I commented on on line 53 and 54 :-) `classic` and `default`

> But I see lots of twenty ten themed around. I wonder if there are any stats online for themes..

The theme pages show the number of active installations, see https://wordpress.org/themes/twentyten/ for example.

Reply:
https://wordpress.org/themes/twentytwentyone/
https://wordpress.org/themes/twentytwenty/
https://wordpress.org/themes/twentynineteen/
https://wordpress.org/themes/twentyseventeen/
https://wordpress.org/themes/twentysixteen/
https://wordpress.org/themes/twentyfifteen/
https://wordpress.org/themes/twentyfourteen/
https://wordpress.org/themes/twentythirteen/
https://wordpress.org/themes/twentytwelve/
https://wordpress.org/themes/twentyeleven/
https://wordpress.org/themes/twentyten/

All of these seem to have very good numbers.

Reply:
Yes. But not `classic` and `default`, which is why I want to remove them :-)

Reply:
> I would say the default and classic themes are not heavily used. 

I agree, they should be removed.